code/game/objects/items/melee/misc.dm

```diff
@@ -131,6 +131,26 @@
 		user.death(FALSE)
 	REMOVE_TRAIT(src, TRAIT_NODROP, SABRE_SUICIDE_TRAIT)
 
+/obj/item/melee/sabre/bee
+	name = "the stinger"
+	desc = "Taken from a giant bee and forget one thousand time in pure honey. Its extremly light and sharp."
```

Comment:
Try this sentence again

Reply:
A PURE JAPANESE HONEY

FORGET OVER 1000 TIME IT HONEY

THEN FORCED TO RELEARN HOW TO BE HONEY

SUPERIOR JAPANESE NT HONEY, STRIKE DOWN ANY WESTERN HONEY LIKE KNIFE THROUGH BUTTER

NO GAIJIN CAN STAND BEFORE JAPAN 1000 HONEY